.gitignore

```diff
@@ -1,8 +1,21 @@
-# Python Bytecode Cache
+# Binaries
```

Comment:
what?
https://stackoverflow.com/questions/16869024/what-is-pycache#16869074

> These are bytecode-compiled and optimized bytecode-compiled versions of your program's files, respectively.

while `Binaries` isn't wrong, why did you change this? i feel like the current one is more specific, but I'm just nitpicking at this point 🦆

Reply:
Well the diff suggested I change that line specifically, but it's more like I rewrote it entirely. The actual term doesn't matter too much.